pkg/controller/controller_test.go

```diff
@@ -761,7 +762,9 @@ func (s *ControllerSuite) TestRuntimeObjEventLogs(c *C) {
 }
 
 func (s *ControllerSuite) TestDeferPhase(c *C) {
-	os.Setenv(kube.PodNSEnvVar, "test")
+	if err := os.Setenv(kube.PodNSEnvVar, "test"); err != nil {
+		log.Printf("Error %v setting env variable", err)
+	}
```

Comment:
In tests, the appropriate thing to do if something "fails" or is unexpected is to fail the test right away. Logging and ignoring the error is certainly the wrong thing to do in this case.

Reply:
@julio-lopez 
```
if err := os.Setenv(kube.PodNSEnvVar, "test"); err != nil {
    t.Fatalf("Error setting env variable: %v", err)
}
```
Would this approach would be fine ?